README.md

```diff
@@ -66,9 +66,9 @@ This is a pretty simple note-taking app, but it's a good example of how you can
 
 ### Relevant code:
 
-- creating users, and logging in and out [./app/models/user.server.ts](./app/models/user.server.ts)
```

Comment:
I think here we're trying to show them from the README where to find the file. So we should keep these.